backstopper-core/src/test/java/com/nike/backstopper/util/ApiErrorUtilTest.java

```diff
@@ -0,0 +1,88 @@
+package com.nike.backstopper.util;
+
+import com.nike.backstopper.apierror.ApiErrorBase;
+import com.nike.internal.util.MapBuilder;
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Map;
+import java.util.UUID;
+
+import static com.nike.backstopper.util.ApiErrorUtil.isApiErrorEqual;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(DataProviderRunner.class)
+public class ApiErrorUtilTest {
+
+    @Test
+    public void constructor_code_coverage() {
+        ApiErrorUtil util = new ApiErrorUtil();
+        assertThat(util).isNotNull();
+    }
+
+    @Test
+    public void generate_api_error_hashcode_generates_expected_hashcodes() {
+        Map<String, Object> metadata = MapBuilder.<String, Object>builder().put("foo", UUID.randomUUID().toString()).build();
```

Comment:
I would duplicate the `Objects.hash(...)` here in the test and verify that the hashcode returned by the method matches the expected `Objects.hash(...)` value. It's a bit of a tautology, but works as a sanity check and can catch if the method gets updated in the future to make sure we take a step back and ask if it's a change we really want.

Reply:
good call. i've updated